animations/implicit_widgets.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: page
+title: "Flutter implicit animations"
+permalink: /animations/implicit_widgets.html
+---
+
+* TOC Placeholder
+{:toc}
+
+
+If you’re new to Flutter or new to animation, consider starting with the built-in Flutter implicit widgets. Implicit animations are useful when you want to quickly add an animation without having to write a lot of animation code because the Flutter SDK implicit widgets already include the `AnimationController`, the ticker provider, the listener, and other animations elements.
+
+Implicit animations are distinct from explicit animations which involve building the animation action using `AnimationController`. And implicit animations are different from transition widgets since you don't need to create an animation widget using `AnimatedWidget`, you simply use an existing implicit animation widget such as `AnimatedOpacity` or `AnimatedIcon`.
+
+
+
+## Using an implicit animation widget
+
+<div>
+<table class="table" width="100%">
+  <col width="35%">
+  <col width="65%">
+	<tbody>
+    <tr>
+      <td><img src="images/implicit.png" alt="Implicit animation widgets"></td>
+      <td>
+      1. Create your <code>StatefulWidget</code>.<br>
+      2. Create the <code>State</code> class to hold the animation object and the implicit animation widget.<br>
+      3. Specify the <code>BuildContext</code> to return the implicit animation widget, such as <code>AnimatedOpacity</code>.<br>
```

Comment:
These three steps can probably be condensed into "placing your implicit animation widget in a stateful widget's build method," since they are just the standard way of creating a stateful widget.